.appveyor.yml

```diff
@@ -13,7 +13,7 @@ install:
   - ps: Install-Product node 8
   - chocolatey install make
   - chocolatey install gnuwin32-coreutils.portable
```

Comment:
Can you add the retry to the two above `chocolatey install` lines as well? They also fail appveyor builds occasionally.

Reply:
Done.